genshinstats/genshinstats.py

```diff
@@ -63,14 +74,11 @@ def set_cookie_auto(browser: str=None) -> None:
     If a specifc browser is set, gets data from that browser only.
     Avalible browsers: chrome, chromium, opera, edge, firefox
     """
-    import browser_cookie3
     logger.debug(f'Loading cookies automatically.')
-    load = getattr(browser_cookie3,browser.lower()) if browser else browser_cookie3.load
-
     # we want to keep user data secure, so don't use any login ticket cookies
-    for c in load(domain_name='.mihoyo'):
-        if c.name in ['ltuid','ltoken']:
-            session.cookies.set(c.name,c.value)
+    cookies=get_browser_cookie(browser)
+    for name,value in cookies.items():
+        session.cookies.set(name,value)
```

Comment:
We can use `session.cookies.update(get_browser_cookie(browser))` here, no need to set every cookie separately.
The comment is no longer needed since it's in the `get_browser_cookie` function.

Reply:
Thanks for pointing that out